src/connection/serverComponent.ts

```diff
@@ -90,79 +88,54 @@ export class ServerComponent {
    */
   public static async checkForUpdate(): Promise<UpdateStatus> {
     let updateResult: UpdateStatus = UpdateStatus.NONE_AVAILABLE;
+
+    const extensionPath = extensions.getExtension(`halcyontechltd.vscode-db2i`).extensionPath;
     const instance = getInstance();
     const connection = instance.getConnection();
 
-    const owner = `ThePrez`;
-    const repo = `CodeForIBMiServer`;
-
     try {
-      const result = await octokit.request(`GET /repos/{owner}/{repo}/releases/tags/${SERVER_VERSION}`, {
-        owner,
-        repo,
-        headers: {
-          'X-GitHub-Api-Version': '2022-11-28'
-        }
-      });
+      const assetPath = path.join(extensionPath, `dist`, SERVER_VERSION_FILE);
+      const assetExists = await exists(assetPath);
 
-      ServerComponent.writeOutput(JSON.stringify(result));
+      ServerComponent.writeOutput(JSON.stringify({assetPath, assetExists}));
 
-      const newAsset = result.data.assets.find(asset => asset.name.endsWith(`.jar`));
-
-      if (newAsset) {
-        const url = newAsset.browser_download_url;
-        const basename = newAsset.name;
+      if (assetExists) {
+        const basename = SERVER_VERSION_FILE;
         const lastInstalledName = Config.getServerComponentName();
 
-        if (lastInstalledName !== basename || this.installed === false) {
-          const updateQuestion = await window.showInformationMessage(
-            `Database Extension update required`, 
-            {
-              modal: true,
-              detail: `An update to the database server component is required: ${basename}`
-            }, 
-            `Update`
-          );
```

Comment:
So here we will always update the server component if there is update available without letting user decide?